docker/README.md

```diff
@@ -2,55 +2,140 @@
 
 ## Prerequisites & Troubleshooting
 
-Before attempting to build openlibrary using the Docker instructions below, please follow this checklist. If you encounter an error, this section may serve as a troubleshooting guide:
+The openlibrary repository must be cloned with `ssh` and *not* `https`. See the [Git Cheat Sheet](https://github.com/internetarchive/openlibrary/wiki/Git-Cheat-Sheet) for more, including how to fix this if you use `git clone https://...`
 
-- Install `docker-ce`: https://docs.docker.com/get-docker/ (tested with version 19.*)
-- Install `docker-compose`: https://docs.docker.com/compose/install/
-- Make sure you `git clone` openlibrary using `ssh` instead of `https` as git submodules (e.g. `infogami` and `acs`) may not fetch correctly otherwise. You can modify an existing openlibrary repository using `git remote rm origin` and then `git remote add origin git@github.com:internetarchive/openlibrary.git`. Then run `git submodule init; git submodule sync; git submodule update` to get rid of the issue.
+Windows users should see [Fix line endings, symlinks, and git submodules](https://github.com/internetarchive/openlibrary/wiki/Git-Cheat-Sheet#fix-line-endings-symlinks-and-git-submodules-only-for-windows-users-not-using-a-linux-vm).
 
-## Setup/Teardown Commands
+Before attempting to build openlibrary using the Docker instructions below, please read through these opening notes about setting up Docker. If you encounter an error, the following section may serve as a troubleshooting guide.
 
-### For Windows Users Only
+### Install Docker Engine or Docker Desktop
+Linux users (including those using a VM), can install Docker Engine or Docker Desktop. Docker has an [FAQ about the difference](https://docs.docker.com/desktop/faqs/linuxfaqs/) between the two. Windows and macOS users should use Docker Desktop as of early 2023.
 
-**Note:** If you get permission issues while executing these commands please run git bash shell as an Administrator.
+- Docker Engine: https://docs.docker.com/engine/install/#server (tested with 19.*)
+- Docker Desktop: https://docs.docker.com/get-docker/
 
-```bash
-# Configure Git to convert CRLF to LF line endings on commit
-git config --global core.autocrlf input
+### A note on `docker-compose` and `docker compose`
+
+As of early 2023, following the installation instructions on Docker's website will install either Docker Desktop, which includes Docker Compose, it will install `docker-ce` and `docker-compose-plugin` (Linux only), both of which obviate the need to install `docker-compose` separately.
```

Comment:
```suggestion
As of early 2023, following the installation instructions on Docker's website will install either Docker Desktop, which includes Docker Compose v2, it will install `docker-ce` and `docker-compose-plugin` (Linux only), both of which obviate the need to install `docker-compose` v1 separately.
```